packages/app/src/sandbox/compile.ts

```diff
@@ -687,41 +688,43 @@ async function compile(opts: CompileOptions) {
         if (htmlModule && htmlModule.code) {
           html = htmlModule.code;
         }
+
         const { head, body } = getHTMLParts(html);
+        if (!lastHeadHTML && !lastBodyHTML) {
```

Comment:
This should prevent us from duplicating html content. Changed it to simply reload on any change to html.